### DISCUSSION TITLE
CVXPY stuck in Compilation step during solving linear program

### DISCUSSION THREAD

**Discussion by namakshenas** [cvxpy/cvxpy#2170] [General]
I'm trying to solve a linear program with 869532 variables and 8869 constraints in CVXPY. The CVXPY gets stuck in the Compilation session right before passing the problem to the solver. Here's the verbose log:

```
-------------------------------------------------------------------------------
                                  Compilation                                  
-------------------------------------------------------------------------------
(CVXPY) Jul 05 11:23:23 AM: Compiling problem (target solver=ECOS).
(CVXPY) Jul 05 11:23:23 AM: Reduction chain: Dcp2Cone -> CvxAttr2Constr -> ConeMatrixStuffing -> ECOS
(CVXPY) Jul 05 11:23:23 AM: Applying reduction Dcp2Cone
(CVXPY) Jul 05 11:23:25 AM: Applying reduction CvxAttr2Constr
(CVXPY) Jul 05 11:23:26 AM: Applying reduction ConeMatrixStuffing
```

After, say half an hour, nothing happens. So, my question is whether there is an option to possibly bypass the reduction or facilitate this process.

PS. I have tried different solvers resulting in no effect.

**Comment by rileyjmurray** [comment]
Hi @namakshenas, thanks for coming by. Can you please post the code used to generate your problem? It's fine if the code uses fake data, as long as the dimensions of all variables and the nature of the constraints are the same as with your "actual" data.

**Reply by namakshenas** [reply]
Hi @rileyjmurray 
Fake data is OK, but can't share the code.

**Reply by rileyjmurray** [reply]
@namakshenas, unfortunately, there's very little we can do to help without seeing code. If you're comfortable sharing code _pivately_ then we can still try to help. You can reach out to us by emailing us at cvxpydevs@gmail.com.

In the meantime, I'll close this discussion thread.
